test/cxx/simple_struct/simple_struct.cc

```diff
@@ -42,7 +50,7 @@ class rec {
 };
 
 int main() {
-  rec testrec( 1, 2 );
+  static rec testrec( 1, 2 );
```

Comment:
`static` was in the Forza repo. In this `devel` repo, we previously only had `map`, `vector` and `vector_obj`. I resolved any differences by choosing the Forza repo's version, since a lot more time was spent on "stl" tests in Forza.